www/js/components/ToggleSwitch.tsx

```diff
@@ -7,9 +7,7 @@ const ToggleSwitch = ({ value, buttons, ...rest }: SegmentedButtonsProps) => {
   return (
     <SegmentedButtons
       value={value as any}
-      onValueChange={(v) => rest.onValueChange(v as any)}
       buttons={buttons.map((o) => ({
-        value: o.value,
```

Comment:
They are just unnecessary because including `value: o.value` does nothing when `...o` is included later anyway.
Same with `rest.onValueChange` and `...rest`

Typescript probably hinted to this once put on a stricter mode